src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -30,6 +30,23 @@
     <Delete Files="@(RoslynFrameworkAssemblies)" />
   </Target>
 
+  <Target Name="PublishNETAnalyzers"
+          BeforeTargets="Build;PublishNetSdk">
+     <PropertyGroup>
+      <AnalyzerAssembliesDirectory>$(ArtifactsBinDir)$(Configuration)\Sdks\Microsoft.NET.Sdk\analyzers</AnalyzerAssembliesDirectory>
+      <AnalyzerTargetsDirectory>$(AnalyzerAssembliesDirectory)\build</AnalyzerTargetsDirectory>
+    </PropertyGroup>
+    <ItemGroup>
+      <AnalyzerAssemblies Include="$(PkgMicrosoft_CodeAnalysis_NetAnalyzers)/analyzers/dotnet/cs/**/*.dll" />
+      <AnalyzerAssemblies Include="$(PkgMicrosoft_CodeAnalysis_NetAnalyzers)/analyzers/dotnet/vb/Microsoft.CodeAnalysis.VisualBasic.NetAnalyzers.dll" />
```

Comment:
It looks like this will probably miss the localized resources for the VB analyzers.

Reply:
There are no VB localized resources. `Microsoft.CodeAnalysis.NetAnalyzers.dll` contains all the resources for both C# and VB and each assembly then uses `Microsoft.CodeAnalysis.NetAnalyzers.resources.dll`